server/controllers/events.js

```diff
@@ -0,0 +1,28 @@
+const models = require('../models/events');
+
+const getAllEvents = async (req, res) => {
+  let limit = req.query.limit || 10;
+  let page = req.query.page || 0;
+  const { rows } = await models.getAllEvents(limit,page);
```

Comment:
Suggest adding try/catch blocks when using async/await so that the API can send a failed HTTP response if the database query fails.